src/strings/PowerFxResources.en-US.resx

```diff
@@ -3291,21 +3291,41 @@
     <comment>function_parameter - Second argument to the Patch function - the updates to be applied to the given rows.</comment>
   </data>
   <data name="AboutRemove" xml:space="preserve">
-    <value>Removes a specific record or records from a data source</value>
+    <value>Removes (optionally All) items from the specified 'collection'.</value>
     <comment>Description of 'Remove' function.</comment>
   </data>
-  <data name="RemoveDataSourceArg" xml:space="preserve">
-    <value>data_source</value>
-    <comment>function_parameter - First parameter for the Remove function. The data source that contains the records that you want to remove from. Translate this string. When translating, maintain as a single word (i.e., do not add spaces).</comment>
+  <data name="RemoveArg1" xml:space="preserve">
+    <value>collection</value>
+  <comment>function_parameter - First parameter of the Remove function - the name of the collection to have an item removed.</comment>
   </data>
-  <data name="RemoveRecordsArg" xml:space="preserve">
-    <value>remove_record(s)</value>
-    <comment>function_parameter - One or more records to be removed. Translate this string. When translating, maintain as a single word (i.e., do not add spaces).</comment>
+  <data name="RemoveArg2" xml:space="preserve">
+    <value>item</value>
+    <comment>function_parameter - Second parameter to the Remove function - the item to be removed.</comment>
   </data>
   <data name="ErrRemoveAllArg" xml:space="preserve">
     <value>If provided, last argument must be 'RemoveFlags.All'. Is there a typo?</value>
     <comment>{Locked=RemoveFlags.All} Error Message, RemoveFlags.All is an enum value that does not get localized.</comment>
   </data>
+  <data name="AboutRemove_collection" xml:space="preserve">
+    <value>The collection to remove rows from.</value>
```

Comment:
>The collection to remove rows from. [](http://example.com/codeflow?start=11&length=35)

Suggestion: "The collection from which rows are to be removed." This avoid ending a sentence with a proposition.